src/core/Timeline.cpp

```diff
@@ -90,7 +90,7 @@ float Timeline::getTempoAtColumn( int nColumn ) const {
 		return m_fDefaultBpm;
 	}
 
-	float fBpm;
+	float fBpm = 120.0;
```

Comment:
Can we make this default to `m_fDefaultBpm`? It might make debugging easier.

Reply:
Sure, that makes sense!